file_browser/src/file_browser/local/resets/_typography.scss

```diff
@@ -0,0 +1,12 @@
+@import "../fonts";
+
+body {
+  margin: 0;
+  font-family: $font;
+  font-size: medium;
+  font-weight: 600;
+}
+
+.app {
+  text-align: center;
+}
```

Comment:
An easy way to apply a font globally, instead of doing this, would be:
```css
* {
  font-family: $font;
}
```

The `*` selector means "every element". You can also move the other properties from body into it.

As for the `margin: 0` in body, this has nothing to do with typography, and therefore should appear in the main file.